buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -115,19 +124,36 @@ public File getJarExpandDir() {
         );
     }
 
-    public void setExcludes(String... classes) {
-        excludes.clear();
+    public void allowMissingClasses(String... classesOrPackages) {
+        for (String each : classesOrPackages) {
+            missingClassExcludes.add(each);
+        }
+    }
+
+    public void allowViolations(String... violatingClasses) {
+        for (String each : violatingClasses) {
+            violationsExcludes.add(each);
+        }
+    }
+
+    public void allowAllMissingClasses() {
```

Comment:
I don't think this is necessary? The method above taking varargs would come through as an empty array if called with no args, so this can be handled there.